pkg/handlers/optimizely_config_test.go

```diff
@@ -8,12 +24,12 @@ import (
 	"testing"
 
 	"github.com/go-chi/chi/v5"
-	"github.com/optimizely/go-sdk/pkg/config"
 	"github.com/stretchr/testify/suite"
 
 	"github.com/optimizely/agent/pkg/middleware"
 	"github.com/optimizely/agent/pkg/optimizely"
 	"github.com/optimizely/agent/pkg/optimizely/optimizelytest"
+	"github.com/optimizely/go-sdk/pkg/config"
```

Comment:
I'm only guessing: Here too do we need /v2/ ?  I only ask because I'm not fully qualified to review this 😆

Reply:
same for all go-sdk module paths 😆